### DISCUSSION TITLE
Draggable container

### DISCUSSION THREAD

**Discussion by cojoclaudiu** [KingSora/OverlayScrollbars#588] [Q&A]
Hello,
Can you drag the whole container similar like https://norserium.github.io/react-indiana-drag-scroll/ ?
Or is there any implementation with https://norserium.github.io/react-indiana-drag-scroll/ I was trying without any result

Thank you

**Comment by Thinkuni-Y** [comment]
Try this hook [react-use-draggable-scroll](https://github.com/rfmiotto/react-use-draggable-scroll) for react.
Just like this:
```javascript
    const isFirstMount = useFirstMountState();
	const scrollbarRef = useRef(null);
    const { events } = useDraggable(
        { current: scrollbarRef.current?.osInstance()?.elements().viewport },
        { isMounted: !isFirstMount },
    );

    return <OverlayScrollbarsComponent ref={scrollbarRef} {...events}></OverlayScrollbarsComponent>;
```

**Comment by slowwriter** [comment]
@cojoclaudiu Do you have any working solution?
